apps/saleor-app-checkout/pages/api/set-payment-provider-settings.ts

```diff
@@ -52,4 +43,5 @@ const handler: NextApiHandler = async (req, res) => {
     return res.status(500).json({ error });
   }
 };
-export default allowCors(requireAuthorization(handler, ["HANDLE_PAYMENTS"]));
```

Comment:
Where do we specify that `HANDLE_PAYMENTS` permission is required for accessing this route?